src/langrila/memory/cosmos.py

```diff
@@ -0,0 +1,45 @@
+from azure.cosmos import exceptions, CosmosClient, PartitionKey
+import logging
+import os
+import uuid
+
+from ..base import BaseConversationMemory
+
+class CosmosConversationMemory(BaseConversationMemory):
+    def __init__(self, endpoint_name: str, key_name: str, dbname_name: str, containername_name: str, pkey_name: str):
+        self.endpoint = os.getenv(endpoint_name)
+        self.key = os.getenv(key_name)
+        self.dbname = os.getenv(dbname_name)
+        self.containername = os.getenv(containername_name)
+        self.pkey = '/' + os.getenv(pkey_name).strip('/')
```

Comment:
What is this used for? and what's the input? (sorry I'm not familiar with cosmosdb)

Reply:
Partition key is used to distribute the data into physical partitions of Cosmos DB backend.
I'm acceptable to change 'pkey' to 'partition_key' to clarify the meaning.

Reply:
I see, but partition_key seems not to be used in this module right now and not always required. Thus it might be better if we can choose whether we use partition_key or not. I wonder we can use partition_key as the argument of `self.container.create_item()`.

Reply:
Sorry, you're right. I forgot to remove 'pkey' during fixing the logic in 'store()'.